nextstrain/cli/command/build.py

```diff
@@ -60,7 +60,9 @@ def register_parser(subparser):
         help    = "Number of CPUs/cores/threads/jobs to utilize at once.  "
                   "Limits containerized (Docker, AWS Batch) builds to this amount.  "
                   "Informs Snakemake's resource scheduler when applicable.  "
-                  "Informs the AWS Batch instance size selection.",
+                  "Informs the AWS Batch instance size selection.  "
+                  "By default, no constraints are placed on how many CPUs are used by a build;"
```

Comment:
Nitpick, but a space at the end of this line would change the resulting text from:

> By default, no constraints are placed on how many CPUs are used by a build;builds may use all that are

to:

> By default, no constraints are placed on how many CPUs are used by a build; builds may use all that are